js-envs/src/main/scala/org/scalajs/jsenv/nodejs/NodeJSEnv.scala

```diff
@@ -66,34 +65,34 @@ class NodeJSEnv private (
   /** Retry-timeout to wait for the JS VM to connect */
   protected val acceptTimeout = 5000
 
-  override def jsRunner(classpath: CompleteClasspath, code: VirtualJSFile,
-      logger: Logger, console: JSConsole): JSRunner = {
-    new NodeRunner(classpath, code, logger, console)
+  override def jsRunner(libs: Seq[ResolvedJSDependency], sjsCode: VirtualJSFile,
+      code: VirtualJSFile): JSRunner = {
+    new NodeRunner(libs, sjsCode, code)
   }
 
-  override def asyncRunner(classpath: CompleteClasspath, code: VirtualJSFile,
-      logger: Logger, console: JSConsole): AsyncJSRunner = {
-    new AsyncNodeRunner(classpath, code, logger, console)
+  override def asyncRunner(libs: Seq[ResolvedJSDependency],
+      sjsCode: VirtualJSFile, code: VirtualJSFile): AsyncJSRunner = {
+    new AsyncNodeRunner(libs, sjsCode, code)
   }
 
-  override def comRunner(classpath: CompleteClasspath, code: VirtualJSFile,
-      logger: Logger, console: JSConsole): ComJSRunner = {
-    new ComNodeRunner(classpath, code, logger, console)
+  override def comRunner(libs: Seq[ResolvedJSDependency],
+      sjsCode: VirtualJSFile, code: VirtualJSFile): ComJSRunner = {
+    new ComNodeRunner(libs, sjsCode, code)
   }
 
-  protected class NodeRunner(classpath: CompleteClasspath,
-      code: VirtualJSFile, logger: Logger, console: JSConsole
-  ) extends ExtRunner(classpath, code, logger, console)
+  protected class NodeRunner(libs: Seq[ResolvedJSDependency],
+      sjsCode: VirtualJSFile, code: VirtualJSFile
+  ) extends ExtRunner(libs, sjsCode, code)
        with AbstractNodeRunner
```

Comment:
Since you're here, you can fix the coding style:
```scala
  protected class NodeRunner(libs: Seq[ResolvedJSDependency],
      sjsCode: VirtualJSFile, code: VirtualJSFile)
      extends ExtRunner(libs, sjsCode, code) with AbstractNodeRunner
```
Same for the two other classes just below.